ui/ui-frontend/projects/vitamui-library/src/assets/shared-i18n/en.json

```diff
@@ -4253,7 +4253,9 @@
       "SHOW_TREE_PARENT": "Show only results",
       "TREE_LEAVES_TITLE": "Attachment position",
       "TREE_TITLE": "Trees and plans",
-      "LOAD_MORE": "Load more"
+      "LOAD_MORE": "Load more",
+      "ORPHANS_NODE": "Without attachment",
+      "KEY_VALUE_NODE": "Attachment order"
```

Comment:
Ce n'est pas la même traduction que le français, est-ce normal ? `With order of attachment` ?

Reply:
J'ai copié d'ARCHIVE_SEARCH vers COLLECT, pas de fr vers en. Après, il est vrai que ça dit pas la même chose entre fr et en, mais c'est pas l'objet de cette PR.

Reply:
Oui enfin si on s'arrête à l'objet de la PR, c'est du clean code; donc ça fait parti du clean code de corriger une mauvaise traduction que l'on remarque ;)

Reply:
Corrigé